Source/WebCore/dom/ElementRareData.cpp

```diff
@@ -40,7 +40,8 @@ struct SameSizeAsElementRareData : NodeRareData {
     void* intersectionObserverData;
     void* typedOMData[2];
     void* resizeObserverData;
-    void* resizeObserverSize;
+    LayoutUnit lastRemembedLogicalWidth;
+    LayoutUnit lastRemembedLogicalHeight;
```

Comment:
I don't know if it's better to be explicit like this, or be more generic like `void* pointers[16];` and just use `LayoutUnit lastRemembedSize[2];` or even `LayoutUnit layoutUnits[2];`. Leaving this for other reviewers.

Reply:
Hmm, yes, `LayoutUnit lastRemembedSize[2]` seems better if we use `LayoutUnit`.